warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -463,12 +506,26 @@ public void map(K1 key, V1 value, Context context) throws IOException, Interrupt
             value.setAuxProperty(ErrorDataTypeHandler.PROCESSED_COUNT, "1");
         }
 
-        // Determine whether the event date is greater than the interval. Excluding fatal error events.
-        if (!value.fatalError() && null != myInterval && 0L != myInterval && (value.getDate() < (now.get() - myInterval))) {
-            if (log.isInfoEnabled())
-                log.info("Event with time " + value.getDate() + " older than specified interval of " + (now.get() - myInterval) + ", skipping...");
-            getCounter(context, IngestInput.OLD_EVENT).increment(1);
-            return;
+        if (!value.fatalError()) {
+            // Determine whether the event date is greater than the interval. Excluding fatal error events.
+            Long myInterval = dataTypeDiscardIntervalCache.get(value.getDataType().typeName());
+            if (null != myInterval && 0L != myInterval && (value.getDate() < (now.get() - myInterval))) {
+                if (log.isInfoEnabled())
+                    log.info("Event with time " + value.getDate() + " older than specified interval of " + (now.get() - myInterval) + ", skipping...");
+                getCounter(context, IngestInput.OLD_EVENT).increment(1);
+                return;
```

Comment:
Looks good.  Perhaps the date type discard interval could be made into a predicate in the future.

Reply:
That is indeed my expectation here.

Reply:
doing it...